pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -1194,3 +1200,15 @@ func envs(envs []corev1.EnvVar, envVar corev1.EnvVar) []corev1.EnvVar {
 	}
 	return envs
 }
+
+// Volume names must be valid RFC 1123 Label Names: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names
+func secretNameToVolumeName(secretName string, volCounter *volumeCounter) string {
+	volumeName := fmt.Sprintf(volumeSecretNameTemplate, secretName)
+	// leave space for - and 3 digits
+	if len(volumeName) > 63 {
+		volumeName = volumeName[:59]
+	}
+	volumeName = fmt.Sprintf("%s-%v", volumeName, volCounter.count)
```

Comment:
Would there be any value in using leading zeros? Like `volume-name-001` instead of `volume-name-1`?

Reply:
can't think of anything